src/schemes/fluid/weakly_compressible_sph/rhs.jl

```diff
@@ -59,6 +59,11 @@ function interact!(dv, v_particle_system, u_particle_system,
                                             particle, neighbor,
                                             m_a, m_b, p_a, p_b, rho_a, rho_b, pos_diff,
                                             distance, grad_kernel, correction)
+        # Add convection term when using `TransportVelocityAdami`
```

Comment:
```suggestion
        
        # Add convection term (only when using `TransportVelocityAdami`)
```
Also added an empty line.